src/resultset/formatters/resultset_replycompact.c

```diff
@@ -163,7 +163,7 @@ void ResultSet_ReplyWithCompactHeader(RedisModuleCtx *ctx, const ResultSetHeader
         RedisModule_ReplyWithArray(ctx, 2);
         Column *c = header->columns[i];
         ColumnTypeUser t;
-        char *identifier = c->alias? c->alias: c->name;
+        char *identifier = c->name;
```

Comment:
Use alias when provided as identifier for line 183, lookup entity using c->name.

Reply:
done